assets/js/script.js

```diff
@@ -0,0 +1,291 @@
+//Part 1 - Shuffle array, add a random number btw 1-4 into array(for adding rounds)
+var instructorMoves = [37,38,39,40]
+var whichPlayer = 0
+var playerTest = 0
+var scoreP1 = 0
+var scoreP2 = 0
+var counterP1 = 0
+var counterP2 = 0
+var rounds = 1
+
+
+function shuffleArray(arr) {
+  for (var i = arr.length - 1; i > 0; i--){
+    var shuffle = Math.floor(Math.random() * (i + 1)) // <- semi-colon needed to seperate from next line
+    ;[arr[i], arr[shuffle]] = [arr[shuffle], arr[i]]
+  }
+  return arr
+}
+
+
+$(function(){
+  var $body = $('body')
+  var $instructor = $('.instructor')
+  var $button = $('button')
+  var $playerOne = $('.playerOne')
+  var $playerTwo = $('.playerTwo')
+  $('.playerOneDiv').addClass("player1")
+  $('.playerTwoDiv').addClass("player2")
+  $('.instructorDiv').addClass("instructorPic")
+  var $resetButton = $('<button class = "resetBtn">Reset</button>')
+  var randSequence = shuffleArray(instructorMoves)
+
+
+  //Part 2a - Add turns after round 1
+  function addTurn(){
+    instructorMoves = instructorMoves.concat(instructorMoves)
+    randSequence = shuffleArray(instructorMoves)
+    return randSequence;
+  }
+  //Making the instructor move according to number sequence
+  function instructorDance () {
+    var instructorMovesPic = [
+      { number: 37, image: 'url("./assets/images/instructor_1.png")' },
+      { number: 38, image: 'url("./assets/images/instructor_2.png")' },
+      { number: 39, image: 'url("./assets/images/instructor_3.png")' },
+      { number: 40, image: 'url("./assets/images/instructor_4.png")' }
+    ];
+    $('.instructorDiv').addClass("move");
+    var timeoutInterval = 1000
+    let test = randSequence.map(function(value) {
+      return (value - 36);
+    })
+
+    for(let e = 0; e <= randSequence.length; e++){
```

Comment:
hmm? why not just tagged a certain class into one div, so you don't have to do this loop instead.